src/components/ATATFileUpload.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div v-cloak 
+    @dragenter="onDragEnter" 
+    @drop.prevent="addDropFile" 
+    @dragover.prevent>
+     <v-file-input
+      ref="atatFileUpload"
+      :id = "id + 'FileUpload'"
+      :class="[{'v-text-field--is-hovering' : isHovering},'atat-file-upload']"
+      multiple
+      prepend-icon=""
+      accept="application/pdf,application/vnd.ms-excel"
+      :truncate-length="truncateLength"
+      :clearable="false"
+      @change="fileUploadChanged"
+    >
+      <template v-slot:prepend-inner>
+        <div class="content text-center">
+          <v-icon class="icon-60 mt-10 mb-5">upload_file</v-icon>
+          <h2>Drag and Drop</h2>
+          <p class="mt-1 mb-4 d-flex justify-center">your file here or 
+            <v-btn 
+              class="link-button inline-text" 
+              @mousedown="fileUploadClicked">
+                browse to upload
+              </v-btn>
+          </p>
+          <p class="mb-9">Use a PDF file with a max size of 10 MB.</p>
+        </div>
+      </template>
+    </v-file-input>
+
+    <h2 class="mt-5" v-if="uploadedFiles.length>0">Files Uploaded</h2>
+    <ul v-for="(file, idx) in uploadedFiles" :key="idx">
+      <li>{{ file.name }}</li>
+    </ul>
+   </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component, Prop, PropSync } from "vue-property-decorator";
+
+@Component({})
+export default class ATATFileUpload extends Vue {
+
+  // refs
+  $refs!: {
+    atatFileUpload: Vue &
+    {
+      errorBucket: string[];
+      errorCount: number;
+    };
+  }
+
+  // props
+  @Prop({default: 15}) private truncateLength!: string;
+  @Prop({default: ""}) private id!: string;
+  @Prop({default: ()=>[]}) private validFileFormats!: string[];
+
+  //data
+  // @PropSync("files", {default: ()=>[]}) private _files: File[] = [];
```

Comment:
Remove commented code? or leaving for later?

Reply:
yes, leaving for later.